test/e2e/e2e_test.go

```diff
@@ -35,19 +35,20 @@ import (
 
 var _ = Describe("Workload cluster creation", func() {
 	var (
-		ctx                 = context.TODO()
-		specName            = "create-workload-cluster"
-		namespace           *corev1.Namespace
-		cancelWatches       context.CancelFunc
-		result              *clusterctl.ApplyClusterTemplateAndWaitResult
-		clusterNamePrefix   string
-		clusterctlLogFolder string
+		ctx                                         = context.TODO()
+		specName                                    = "create-workload-cluster"
+		namespace, gkeNamespace                     *corev1.Namespace
+		cancelWatches, gkeCancelWatches             context.CancelFunc
+		result                                      *clusterctl.ApplyClusterTemplateAndWaitResult
+		clusterNamePrefix                           string
+		clusterctlLogFolder, gkeClusterctlLogFolder string
 	)
 
 	BeforeEach(func() {
 		Expect(e2eConfig).ToNot(BeNil(), "Invalid argument. e2eConfig can't be nil when calling %s spec", specName)
 		Expect(clusterctlConfigPath).To(BeAnExistingFile(), "Invalid argument. clusterctlConfigPath must be an existing file when calling %s spec", specName)
 		Expect(bootstrapClusterProxy).ToNot(BeNil(), "Invalid argument. bootstrapClusterProxy can't be nil when calling %s spec", specName)
+		Expect(bootstrapGKEClusterProxy).ToNot(BeNil(), "Invalid argument. bootstrapGKEClusterProxy can't be nil when calling %s spec", specName)
```

Comment:
I'm not sure what this is (yet).  Is this required if we use kind for the bootstrap cluster (instead of GKE)?